content/events/16-17/LAN.md

```diff
@@ -9,7 +10,18 @@ locationlink: Therminal,Ghent Belgium
 description: Zeus' vijfde LAN-Party
 ---
 
-De vijfde editie van Zeus WPI's LAN-Party komt eraan!
-Hou alvast de datum vrij!
+# It's that time of year again
 
-Wij houden jullie op de hoogte!
+Je leest het goed, op **4 november** vindt onze vijfde editie van onze jaarlijkse LAN-Party plaats in **[Studentenhuis De Therminal](http://student.ugent.be/)** (Hoveniersberg 24, 9000 Gent).
+
+Vanaf **16:00** kunnen jullie elkaars Nexus vernietigen in League of Legends, een Ancient ten val brengen in Dota 2, bommen ontmantelen in Counter Strike: Global Offensive, op ontzagwekkende snelheden door de maps van Trackmania rijden, en nog veel, veel meer.
+
+# Inschrijven is verplicht!
+
+Door je in te schrijven krijgen wij een beter beeld van hoeveel volk we kunnen verwachten, wat we allemaal moeten voorzien, en welke competities we kunnen organiseren. **[Klik hier om je in te schrijven.](https://goo.gl/forms/XwHhYB4NkOO2GDoE3)**
+
+# Praktisch
+
+De deuren openen om 16:00. We vragen €2 inkom die gebruikt wordt voor de organisatie van het evenement, alsook prijzengeld. Zeus doelt niet op winst met "Geeks, Attack!".
```

Comment:
We vragen €2 inkom die gebruikt wordt voor de organisatie van het evenement, alsook prijzengeld. Zeus doelt niet op winst met "Geeks, Attack!".
->
We vragen €2 inkom, die gebruikt zal worden om de prijzenpot van onze competities aan te dikken.

ik vind de "zeus doelt niet op winst" overbodig, maar de aandacht vestigen op de competities is belangrijk